MMR.Randomizer/Randomizer.cs

```diff
@@ -847,6 +849,57 @@ private void RandomizeItems()
             _randomized.ItemList = ItemList;
         }
 
+        /// <summary>
+        /// Places plando items in the randomization pool
+        ///   re-implements PlaceItem for plando
+        ///   reasons: we need pass a different list of checks, but itemPool still needs to be decremented
+        ///            we want a different error, stating plando failed not just any randomization
+        ///            for plando, we want the ability to bypass logic
+        /// </summary>
+        /// <param name="itemPool"></param>
+        private void PlacePlandoItems(List<Item> itemPool)
+        {
+            List<PlandoItemCombo> plandoItemCombos = PlandoUtils.ReadAllItemPlandoFiles(itemPool);
+            if (plandoItemCombos != null)
+            {
+                foreach (PlandoItemCombo itemCombo in plandoItemCombos)
+                {
+                    PlandoItemCombo cleaned = PlandoUtils.CleanItemCombo(itemCombo, Random, itemPool, ItemList);
+                    if (cleaned == null)
+                    {
+                        throw new Exception("Error: Plando failed to build with this seed: " + itemCombo.Name);
+                    }
+
+                    int drawCount = 0;
+                    for(int itemCount = 0; drawCount < cleaned.ItemDrawCount && itemCount < cleaned.ItemList.Count; itemCount++)
+                    {
+                        /// for all items, attempt to add, count successes
+                        Item item = cleaned.ItemList[itemCount];
+                        foreach (Item check in cleaned.CheckList)
+                        {
+                            if (itemCombo.SkipLogic || CheckMatch(item, check))
```

Comment:
SkipLogic shouldn't ignore the ForbiddenStartingItems check. There are technical reasons that some items aren't allowed to be starting items, not even for plandos.

Reply:
Changed so that if logic is skipped, we manually check for forbiddenstartingitems like CheckMatch does.